doc/classes/RenderingServer.xml

```diff
@@ -4450,6 +4450,7 @@
 			Sort particles based on their lifetime.
 		</constant>
 		<constant name="PARTICLES_DRAW_ORDER_REVERSE_LIFETIME" value="2" enum="ParticlesDrawOrder">
+			Sort particles based on the inverse of their lifetime.
```

Comment:
Same as above.

Reply:
@Anutrix This hasn't been updated to match the others.